File: Actividad 2/actividad224.java
import java.util.Scanner;

public class actividad224 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Ingrese el peso de la esfera A: ");
        double esferaA = scanner.nextDouble();
        System.out.print("Ingrese el peso de la esfera B: ");
        double esferaB = scanner.nextDouble();
        System.out.print("Ingrese el peso de la esfera C: ");
        double esferaC = scanner.nextDouble();

        if (esferaA > esferaB && esferaA > esferaC) {
            System.out.println("Esfera A es más pesada.");
        } else if (esferaB > esferaA && esferaB > esferaC) {
            System.out.println("Esfera B es más pesada.");
        } else {
            System.out.println("Esfera C es más pesada.");
        }
    }
}

File: Actividad 2/actividad224.py
esferaA = float(input("Ingrese el peso de la esfera A: "))
esferaB = float(input("Ingrese el peso de la esfera B: "))
esferaC = float(input("Ingrese el peso de la esfera C: "))

if esferaA > esferaB and esferaA > esferaC:
    print("Esfera A es mas pesada")
elif esferaB > esferaA and esferaB > esferaC:
    print("Esfera B es mas pesada")
else:
    print("Esfera C es mas pesada")